library/asb_encode_binding.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
```

Comment:
Since we have these docs already, would be nice to pull them out (automatically eventually) and add them to the README.